common/brave_features.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/common/brave_features.h"
+#include "build/build_config.h"
+
+namespace features {
+
+#if defined(OS_ANDROID)
+//  Flag for Brave Rewards.
+#if defined(ARCH_CPU_X86_FAMILY) && defined(OFFICIAL_BUILD)
```

Comment:
Jus curious why this is needed?

Reply:
This is the condition we used in current production, maybe @SergeyZhukovsky knows more context about this.